toolkit/tools/srpmpacker/srpmpacker.go

```diff
@@ -67,7 +67,7 @@ const (
 
 const (
 	defaultBuildDir    = "./build/SRPMS"
-	defaultWorkerCount = "10"
+	defaultWorkerCount = "80"
```

Comment:
Does this change and the above `100` affect our total time ?
I am wondering if there is a chance to get rate-limited by the blobstore ?

Reply:
I've never managed to saturate the connection on my dev box. I've also got a more advanced change that scales this to the available cores but this is a stopgap to help speed up the new parallel builds in the meantime.

Reply:
Superseded by the semaphore PR, reverting.
```suggestion
	defaultWorkerCount = "10"
```